### PR TITLE
In-order tree traversal using python 

### PR DESCRIPTION
**Pull request by rutujadhanawade**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

### Description

Added In-order tree traversal Iterative program using python.  #415 
please review this @Abhijit2505 

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

#415 (Traversal in binary Tree)

### CODE REVIEW

**Comment by rutujadhanawade** [comment]
@atarax665 @plazzy99 @Kajol-Kumari  will you please review this?, this PR is a part of CH20

File: Python/ds/Tree_Iterative_InOrder_Traversal.py [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+		    	curr = curr.left
+		    else:
+			# else if current node is null, we pop an element from the stack,print it and finally set current node to its right child
+			    curr = stack.pop()
+			    print(curr.data, end=' ')
+			    curr = curr.right
+
+
+NodesArray = list(input().split(' '))
+InOrdertree = Tree()
```
Comment (vatsalkesarwani12) [inline Python/ds/Tree_Iterative_InOrder_Traversal.py:54]: add an input message

**Review by vatsalkesarwani12 — CHANGES_REQUESTED** [review]
do requested changes

**Comment by rutujadhanawade** [comment]
> do requested changes

I have added Input message! please review this. @plazzy99

**Review by Kajol-Kumari — APPROVED** [review]
The code is working fine and LGTM!

**Review by vatsalkesarwani12 — APPROVED** [review]
LGTM :)

**Comment by rutujadhanawade** [comment]
@atarax665 Will you please review this .
@Kajol-Kumari and @plazzy99 Thank you for reviewing this PR.

**Review by atarax665 — APPROVED** [review]
Voila! Code looks fine to me. :100:

**Comment by iamrajiv** [comment]
@all-contributors please add @rutujadhanawade for code

**Pull request merged**